projects/ngx-quill/src/lib/quill.service.ts

```diff
@@ -27,9 +32,11 @@ export class QuillService {
       // this means the `zone.js` is not imported.
       // The `__zone_symbol__addEventListener` is basically a native DOM API, which is not patched by zone.js, thus not even going
       // through the `zone.js` task lifecycle. You can also access the native DOM API as follows `target[Zone.__symbol__('methodName')]`.
-      // eslint-disable-next-line @typescript-eslint/dot-notation
-      this.document.addEventListener = this.document['__zone_symbol__addEventListener'] || this.document.addEventListener
-      const quillImport = await import('quill')
+      this.document.addEventListener =
+        // eslint-disable-next-line @typescript-eslint/dot-notation
+        this.document['__zone_symbol__addEventListener'] ||
+        this.document.addEventListener
+      const quillImport = await import(/* webpackChunkName: 'quill' */ 'quill')
```

Comment:
is the comment really needed for it?

Reply:
Not really required. I thought of adding the chunk name because it's normally `[hash].js`. It's easy to search through chunks when it's named when you have say 50+ or 100+. I can remove the chunk name.

Reply:
I was also wondering if there's a plan to let users define the loading mechanism for Quill:
```ts
ConfigModule.forRoot({
  quillLoader: () => import('quill')
})
```

Or:

```ts
ConfigModule.forRoot({
  // Loads module through SystemJS
  quillLoader: () => System.import('quill')
})
```

Or:
```ts
ConfigModule.forRoot({
   // Loads the `<script>...` which exposes Quill constructor on `window`.
  quillLoader: () => loadScript().then(() => window.Quill)
})
```

Reply:
in general there is not such plan. if you think it is something that would be used, feel free to add.